include/arch/arc/v2/aux_regs.h

```diff
@@ -110,6 +110,17 @@
 #define _ARC_V2_FPU_DPFP1H 0x303
 #define _ARC_V2_FPU_DPFP2L 0x304
 #define _ARC_V2_FPU_DPFP2H 0x305
+#define _ARC_V2_MPU_EN 0x409
+#define _ARC_V2_MPU_RDB0 0x422
+#define _ARC_V2_MPU_RDP0 0x423
+#define _ARC_V2_MPU_INDEX 0x448
+#define _ARC_V2_MPU_RSTART 0x449
+#define _ARC_V2_MPU_REND 0x44A
+#define _ARC_V2_MPU_RPER 0x44B
+#define _ARC_V2_MPU_PROBE 0x44C
+
+/* Default reset value for _ARC_V2_MPU_EN register */
+#define ARC_MPU_EN_RESET_VALUE 0x400181C0
```

Comment:
Have some doubt here: I wish to put this in arc_mpu.h, but the header file is not included in assembly code.

Reply:
There's no need to move that default value which is used only one in 1 place into generic header. And that's what I meant in my initial commit - move that definition right in the place where it is used. It's OK to have it done that way:
```c
...
// This is very important definition used for XXX
#define MY_IMPORTNAT_DEFINITION 0x12345678
    super_secret_password = MY_IMPORTNAT_DEFINITION;
...
```